docs/api/text/tsne.py

```diff
@@ -25,17 +25,13 @@ def tsne(docs, labels, outpath, **kwargs):
     tfidf = TfidfVectorizer()
 
     docs   = tfidf.fit_transform(corpus.data)
-    labels = corpus.target
+    target = corpus.target
 
     # Whole corpus visualization
-    tsne(docs, labels, "images/tsne_all_docs.png")
-
-    # Partial corpus visualization
-    # Only visualize the sports, cinema, and gaming classes
-    tsne(docs, labels, "images/tsne_limit_classes.png", classes=['sports', 'cinema', 'gaming'])
```

Comment:
This file generates figures for the documentation; had to remove the partial corpus visualization, but included an example that uses the `labels` arg to name the documents in the target-less fit.

Reply:
It might be obvious but I suggest you include a docstring that states as you put it, "This file generates figures for tnse documentation"